packages/atomic/package.json

```diff
@@ -39,7 +39,8 @@
     "npm:publish:alpha": "node ../../scripts/deploy/publish.js alpha",
     "storybook": "npm run storybook:lit-analyze && start-storybook -s ./dist -p 6006",
     "build:storybook": "npm run storybook:lit-analyze && build-storybook && ncp dist/ storybook-static/",
-    "storybook:lit-analyze": "lit-analyzer dist/types/components.d.ts src/components/**/*.stories.tsx"
+    "storybook:lit-analyze": "lit-analyzer dist/types/components.d.ts src/components/**/*.stories.tsx",
+    "doc:listassets": "node scripts/list-assets.js"
```

Comment:
a bit annoyed by the `listassets` casing here, perharps `doc:list:assets` or `doc:list-assets`

Reply:
I like the dash